test/test_hyperparameters.py

```diff
@@ -464,29 +463,27 @@ def test_normalint(self):
             "param, Type: NormalInteger, Mu: 0 Sigma: 10, Default: 0, Q: 2",
             str(f2_large_q))
 
-        with pytest.warns(RuntimeWarning, match='divide by zero encountered in log'):
-            f3 = NormalIntegerHyperparameter("param", 0, 10, log=True)
-        with pytest.warns(RuntimeWarning, match='divide by zero encountered in log'):
-            f3_ = NormalIntegerHyperparameter("param", 0, 10, log=True)
+        f3 = NormalIntegerHyperparameter("param", 0, 10, log=True)
+        f3_ = NormalIntegerHyperparameter("param", 0, 10, log=True)
```

Comment:
I'm okay with removing the warning check as it's not something we explicitly do, i.e. emit a warning. Just wanted to ask if the warning is still raised however or was this handled in some other way such that no warning happens?

Reply:
This has to do with the fact that I changed what the parameters mu and sigma actually mean for a logged normal variable.

Previously, the mean value would be defined as it it was a Normal, and not a LogNormal. The default value would be defined as log(mu), i.e. log(0) --> the warning occurs. This was simply a bug, as it was not correctly defined.

Now, the parameters work as they do in a LogNormal distribution. As such, the user enterfing mu = 1 -->  mean of distribution at e. While this is a bit unintuitive, I would argue that any other solution is to some extent incorrect and leaves the user guessing. 

As for the warning, it simply never occurs now when the definition is changed.